### DISCUSSION TITLE
Year is 6 digits, can it be 4?

### DISCUSSION THREAD

**Discussion by rexforab** [wojtekmaj/react-daterange-picker#236] [Q&A] [answered]
Probably a silly question, but it seems the mask for the year allows 6 digits.  This can be seen on the online demo page.  Can that be changed with a property or setting to be limited to four digits?

**Comment by wojtekmaj** [comment, marked as answer]
Sure! Technically, year 123456 is totally valid, only in a very distant future. Just set maxDate to any Date with 4-digit year you like, and it will accept 4 digits only :)
